### PR TITLE
Minimap icons

### PR DESCRIPTION
**Pull request by carlosrpg**

### CODE REVIEW

**Comment by carlosrpg** [comment]
I'll need to some to test this with players on both teams to verify if the camps visibility is not shared and follow the expected results

**Comment by carlosrpg** [comment]
I'll probably not have more time to OAA until mid/end Jan so you you guys had code reviews could you please push it to the PR?

**Comment by warpdragon** [comment]
SeemsGood!

File: game/scripts/addon_hud_textures.txt [hunk 1]
```diff
@@ -2,53 +2,69 @@
...
-            "file"     "materials/vgui/hud/minimap_cyan.vmat"
-            "x"        "0"
-            "y"        "0"
-            "width"    "64"
-            "height"   "64"
+          "file"		"materials/vgui/hud/minimap_boss_tier4.vmat"
+          "x"		"0"
+          "y"		"0"
+          "width"		"16"
+          "height"		"16"
```
Comment (carlosrpg) [inline game/scripts/addon_hud_textures.txt]: this is wrong, but I'm not sure I'll be able to fix this.
should have been 32px for all bosses icons

**Comment by warpdragon** [comment]
@carlosrpg fixed it for ya

**Pull request merged**